boring-sys/patches/boring-pq.patch

```diff
@@ -3957,18 +4343,39 @@ index cafae9d17..074ac5906 100644
 -// there are trailing bytes in |in|.
 -OPENSSL_EXPORT int KYBER_parse_private_key(
 -    struct KYBER_private_key *out_private_key, CBS *in);
--
++struct KYBER512_public_key {
++  uint8_t opaque[KYBER512_PUBLIC_KEY_BYTES];
++};
++struct KYBER768_public_key {
++  uint8_t opaque[KYBER768_PUBLIC_KEY_BYTES];
++};
++struct IPDWING_public_key {
++    struct KYBER768_public_key m;
++    uint8_t x[32];
```

Comment:
nit: I find it odd that the curve point is named `x` here but named `xpub` as part of the private key. In the private key, `x` is the name of the secret scalar. Could we go with `xpub` here instead so that we have one name for each thing?